docs/source/operators/deploy-kubernetes.md

```diff
@@ -15,248 +17,495 @@ The following sample kernel specifications apply to Kubernetes deployments:
 
-## Deploying Enterprise Gateway on Kubernetes
+We are using helm templates to manage Kubernetes resource configurations, which allows an end user to easily customize their Enterprise Gateway deployment.
+
+There are two main deployment scenarios if RBAC is enabled in your Kubernetes cluster:
+
+1. Deployment user has **_Cluster Administrator Access_**. In this scenario, you have full access to the cluster and can deploy all components as needed.
+2. Deployment user has **_Namespace Administrator Access_**. This is typical for shared multitenant environments where each Team has control over their namespace, but not the cluster. In this scenario, your cluster Administrator can deploy the RBAC resources and Kernel Image Puller and you can deploy Enterprise Gateway.
+
+## Prerequisites
 
-Once the Kubernetes cluster is configured and `kubectl` is demonstrated to be working on the primary node, it is time to deploy Enterprise Gateway. There are a couple of different deployment options - using helm charts or kubectl templates.
+- Install and configure [kubectl](https://kubernetes.io/docs/tasks/tools/) and [helm3](https://helm.sh/docs/intro/install/) on your workstation.
+- Create the kubernetes namespace where you want to deploy `enterprise-gateway`, for example:
+  ```sh
+  kubectl create namespace enterprise-gateway
+  ```
+- If you use RBAC, you will need cluster Admin access to configure RBAC resources
+- If you plan to use Private docker registry, you will need to have credentials (see configuration steps below)
 
-### Deploying with helm charts
+Once the Kubernetes cluster is configured and `kubectl` is demonstrated to be working, it is time to deploy Enterprise Gateway. There are a couple of different deployment options - using helm or kubectl.
 
-Choose this option if you want to deploy via a [helm](https://helm.sh/) chart.
-If Ingress is desired see [this section](#setting-up-a-kubernetes-ingress-for-use-with-enterprise-gateway) before deploying
-with helm.
+## Deploying with helm
 
-#### Create the Enterprise Gateway kubernetes service and deployment
+Choose this option if you want to deploy via a [helm](https://helm.sh/) chart. You can customize your deployment using value files - review Configuraation section below for details.
 
-Ensure the `enterprise-gateway` namespace exists:
+### Create the Enterprise Gateway kubernetes service and deployment
+
+You can execute the helm command from the checked-out release of the Enterprise Gateway git [repository](https://github.com/jupyter-server/enterprise_gateway.git):
 
 ```bash
-kubectl create namespace enterprise-gateway
+helm  upgrade --install  enterprise-gateway \
+  jupyter-server/enterprise_gateway/etc/kubernetes/helm/enterprise-gateway \
```

Comment:
In this case, I think we should assume the user's CWD is _in_ the EG repo directory...
```suggestion
  etc/kubernetes/helm/enterprise-gateway \
```